core/modules/views/plugins/views_plugin_localization_core.inc

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @file
+ * Definition of views_plugin_localization_core.
+ */
+
+/**
+ * Localization plugin to pass translatable strings through t().
+ *
+ * @ingroup views_localization_plugins
+ */
+class views_plugin_localization_core extends views_plugin_localization {
+
+  /**
+   * Translate a string.
+   *
+   * @param string $string
+   *   The string to be translated.
+   * @param array $keys
+   *   An array of keys to identify the string. Generally constructed from
+   *   view name, display_id, and a property, e.g., 'header'.
+   * @param string $format
+   *   The input format of the string. This is optional.
+   */
+  public function translate_string($string, $keys = array(), $format = '') {
+    return t($string);
+  }
+
+  /**
+   * Save a string for translation.
+   *
+   * @param string $string
+   *   The string to be translated.
+   * @param array $keys
+   *   An array of keys to identify the string. Generally constructed from
+   *   view name, display_id, and a property, e.g., 'header'.
+   * @param string $format
+   *   The input format of the string. This is optional.
+   */
+  public function save_string($string, $keys = array(), $format = '') {
+    // Find a language that is not "en" (if available) to trigger an entry
+    // in module locale's db tables, even if current language is "en".
+    $langcode = 'en';
+    $languages = language_list();
```

Comment:
Do we want this to only fetch enabled languages? This function changed from D7. If so then you'll need `language_list(TRUE)`.

Reply:
Interesting question. What would be the benefit of only fetching enabled languages?

Locale allows translation even to disabled languages. And D7 Views also fetches all.

Reply:
In D7 it fetched all "installed" languages by default https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/language_list/7.x and in Backdrop it fetches all "configured" languages by default https://api.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/language_list/1 but allows for getting just enabled languages. That's useful in cases like when selecting which language content should be in. I wasn't sure if it was needed here so it's probably fine as is.